docs/vspec.md

```diff
@@ -385,6 +386,32 @@ Will also generate non-payload const attributes such as unit/datatype. Default i
 Add additional fields to the nodes in the graphql schema. use: <field_name>,<description>.
 Can be used more than once.
 
+## APIGEAR exporter notes
```

Comment:
This should better be added in a separate file like `apigear.md` in this directory. In the PR #382 a list referencing to other docs will be added, and when that one is merged this part can be moved to a new file.

Reply:
Extracted to a separate file.